src/ElasticSan/test/Get-AzElasticSanPrivateEndpointConnection.Tests.ps1

```diff
@@ -0,0 +1,29 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Get-AzElasticSanPrivateEndpointConnection'))
```

Comment:
Is there any test cover add/update ElasticSan group with PE?

Reply:
Not in the unit tests of ElasticSan, since we're not using the ElasticSan PE cmdlets.

Reply:
Then will you add the test case for get ElasticSan with PE?

Reply:
This pr is ready to review right?

Reply:
@Nickcandy  Yes it's ready